src/greeter/CMakeLists.txt

```diff
@@ -1,3 +1,14 @@
+if(QT_MAJOR_VERSION EQUAL "5")
+    # Keep the unversioned name for Qt5. When upgrading SDDM, the old daemon
+    # might still be running and only know about "sddm-greeter". Keeping the
+    # previous name around also helps users calling it directly.
+    set(GREETER_TARGET sddm-greeter)
```

Comment:
Why not just handle the unversioned name via symlink for this purpose? I'd rather us just change to versioned names unless we intend to quickly drop Qt5 support.

Reply:
What's the benefit for using a symlink instead?

I decided against it because it's one more file and installing a symlink with cmake needs a custom command...

Reply:
It does? `file(CREATE_LINK)` has been available since CMake 3.14...

Reply:
Would need to check whether bumping the requirement would be ok.

Reply:
`set(CMAKE_CXX_STANDARD 17)` was added in CMake 3.20, so you already require that as a minimum.

Reply:
That was most likely an oversight, but nobody complained so it's probably ok :shrug:

Reply:
> set(CMAKE_CXX_STANDARD 17) was added in CMake 3.20, so you already require that as a minimum.

That was actually in CMake 3.8: https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD

Reply:
Oh welp, right, C++20 was in CMake 3.20. My bad.